Source/WTF/wtf/CurrentTime.cpp

```diff
@@ -65,11 +65,15 @@
 #include <glib.h>
 #endif
 
-#if OS(MORPHOS) || OS(AMIGAOS)
+#if OS(MORPHOS)
 #include <exec/system.h>
```

Comment:
Presumably this did not work since there is no such file in AmigaOS?

Reply:
Exactly, so I removed that dependency. Also that header files has a few defines that are MorphOS only, I guess.